distributed/protocol/compression.py

```diff
@@ -23,7 +27,13 @@
 
 from ..utils import ensure_bytes
 
-compressions = {None: {"compress": identity, "decompress": identity}}
+if TYPE_CHECKING:
+    from typing_extensions import Literal
```

Comment:
No, because we are running everything through pre-commit. typing_extensions shouldn't be in the env files for the same reason flake8, black, isort and mypy aren't.

Reply:
To clarify: if you look at #5348, typing_extensions is installed there exclusively in the mypy virtualenv by .pre-conmmit-config.yaml.

Reply:
That makes sense, thanks for clarifying